lib/auth/keystore/pkcs11.go

```diff
@@ -47,6 +47,7 @@ type pkcs11KeyStore struct {
 	log       *slog.Logger
 	isYubiHSM bool
 	semaphore chan struct{}
+	hash      crypto.Hash
```

Comment:
let's make it clear this is the hash to use with newly generated decryption keys